homeassistant/components/climate/__init__.py

```diff
@@ -290,13 +247,18 @@ def target_humidity(self):
         return None
 
     @property
-    def current_operation(self):
-        """Return current operation ie. heat, cool, idle."""
+    def hvac_state(self):
+        """Return current hvac operation ie. heat, cool mode"""
+        return None
```

Comment:
Not sure you where quite finished here. But now it's somewhat confusing. We have hvac_[state] hvac_[modes] and [current_operation] quite unclear how they relate. Also comments make use of "current" clause.

Reply:
Yeah, need to fix the comment and update later the developer page. current_operation has no effect on everything. That was a big wish on architecture that you can see the current action of device on UI.

Reply:
That is also the reason why I don't update it to hvac to avoid that. Also, only 10% of devices support that feature.

Reply:
That only 10% support it now doesn't mean only 10% can support it. It was not an allowed behaviour before.